api/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+repositories {
+        mavenCentral()
+        mavenLocal()
+}
+
+test {
+    useJUnitPlatform()
+}
+
+dependencies {
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: '5.7.1'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.7.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
```

Comment:
please keep the same structure in dependencies. Use either short or full declaration for all